apps/api/src/routes/admin.routes.ts

```diff
@@ -695,6 +695,10 @@ export function configureAdminRoutes(
    *                 type: string
    *                 description: URL to the user's profile image
    *                 example: "https://example.com/user-image.jpg"
+   *               userMetadata:
```

Comment:
the frontend needs to be able to add user metadata in order to keep the user creation APIs 1:1 for SIWE:admin